app/views/admin/api/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="admin">
+  <h2 class="heading"><%= ts('API Tokens') %></h2>
+
+  <ul class="navigation actions" role="navigation">
+    <li><%= span_if_current ts('Create a Token'), new_admin_api_path %></li>
+  </ul>
+
+  <table>

```

Comment:
For screen readers, this needs a summary attribute that describes the table's contents and a caption element, e.g.

```
<table summary="<%= ts("Existing API tokens along with the dates they were created and updated and options for editing them.") %>">
  <caption><%= ts("API Tokens") %></caption>
```